### ISSUE TITLE
Commands does not work and returns `ERR Failed to start`.

### ISSUE THREAD

**Issue by runkecheng** [alibaba/kt-connect#282]
**Describe the bug**

Execute `ktctl`, echo `ERR Failed to start`, no other prompt information.

```
$ ktctl
NAME:
   KtConnect

USAGE:
   ktctl [global options] command [command options] [arguments...]

VERSION:
   0.3.1

AUTHOR:
   Alibaba Cloud

COMMANDS:
...
9:59AM ERR Failed to start
```

**Log**

**Environment (please complete the following information):**

 - OS: ubuntu20.04
```
cat /proc/version
Linux version 5.10.60.1-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Wed Aug 25 23:20:18 UTC 2021
```
 - Kubernetes 
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.3", GitCommit:"c92036820499fedefec0f847e2054d824aea6cd1", GitTreeState:"clean", BuildDate:"2021-10-27T18:41:28Z", GoVersion:"go1.16.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.5", GitCommit:"aea7bbadd2fc0cd689de94a54e5b7b758869d691", GitTreeState:"clean", BuildDate:"2021-09-15T21:04:16Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"linux/amd64"}
```
 - KT Version 
```
ktctl --version
KtConnect version 0.3.1
```

**Additional context**

**Comment by linfan** [comment]
You could ignore this error currently, just put a subcommand, e.g. `ktctl connect`, then the error message will gone.

That message actually means "Ktcal failed to start because no subcommand is provided", but this situation is not handled properly, with be fixed in next release : P

Thanks for the finding.

**Comment by linfan** [comment]
fixed in 0.3.2
